tester/Makefile

```diff
@@ -33,8 +33,8 @@ CFLAGS := -Werror -Wundef \
 	-DST_TESTER \
 	-DSEQUENCE_TRANSFORM_DEBUG=1 \
 	-DSEQUENCE_TRANSFORM_ENHANCED_BACKSPACE=1 \
-	-DSEQUENCE_TRANSFORM_RULE_SEARCH=1 \
-	-DSEQUENCE_TRANSFORM_FALLBACK_BUFFER=1 \
+	-DSEQUENCE_TRANSFORM_RULE_SEARCH=0 \
+	-DSEQUENCE_TRANSFORM_FALLBACK_BUFFER=0 \
```

Comment:
Tester needs these enabled.

Reply:
I need them disabled for testing this branch at the moment, and it's much easier to switch branches of these settings are committed.

I'll roll this back at the end when this is no longer a draft